src/domain/event/__tests__/EventSummaryPage.test.tsx

```diff
@@ -268,3 +278,32 @@ it('shows upcoming and past occurrences', async () => {
     screen.getByRole('heading', { name: 'Menneet tapahtuma-ajat 2 kpl' })
   ).toBeInTheDocument();
```

Comment:
perhaps here could be added that for event2 that is not full, the `enrolmentFull` class and text are not in the document

Reply:
improved tests